apiV1.yaml

```diff
@@ -1185,6 +1185,10 @@ paths:
                 description: Must be a language supported by Grottocenter (see GET /languages route) using the 3-letters format.
                 type: string
                 example: 'eng'
+      - name: precision
+        in: query
```

Comment:
@Clm-Roig Pourrais tu ajouter une description du type "GPS precision in meters"

Reply:
c'est fait !